main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -1830,20 +1830,11 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
       SP ~ capture(optional("_") ~ oneOrMore(Letters.OperatorLetter)) ~ SP ~> Name.Ident
     }
 
-    /**
-      * Namespaces are lower or uppercase.
-      */
-    // TODO NS-REFACTOR remove
-    def Namespace: Rule1[Name.NName] = rule {
-      SP ~ oneOrMore(UpperCaseName).separatedBy("/" | ".") ~ SP ~>
-        ((sp1: SourcePosition, parts: Seq[Name.Ident], sp2: SourcePosition) => Name.NName(sp1, parts.toList, sp2))
-    }
```

Comment:
Should this also be entirely removed? The comment on ns-refactor wants it removed, but the `Names.DotSeparated` rule allows for both `UppercaseName` and `LowerCaseName` so they are not entirely equivalent. Maybe @mlutze can comment on what is okay here?

Reply:
I think we should have something in `Weeder` to check that the names in a namespace are all uppercase.

Reply:
As in, we have it, or we should have it?

Reply:
Should